generators/addons/index.js

```diff
@@ -131,9 +131,18 @@ module.exports = class extends Generator {
                     }
 
                 }
+                if(this.options.ci !== undefined) {
+                    newPkgConfig.scripts.test = "./node_modules/.bin/jest --config ./config/jest.config.json"
+                    newPkgConfig.scripts["test:watch"] = "./node_modules/.bin/jest --config ./config/jest.config.json --watchAll"
+                }
 
             }
-
+            if(this.options.ci !== undefined) {
```

Comment:
@StfBauer here I should probably check if jest also has been selected by user, how can I do that?